CMakeLists.txt

```
@@ -6,6 +6,7 @@ set( CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake )
 
 find_package( jwt-cpp REQUIRED )
 find_package( CURL REQUIRED )
+find_package(UUID REQUIRED)
```

```suggestion
find_package( UUID REQUIRED )
```

(just to match whitespace of previous lines)